src/api/topology-viewer-api.js

```diff
@@ -5,52 +5,83 @@ const API = getAxtionsInstace();
 
 export const structureNode = {
   name: 'sources',
+  topologyViewNode: {
+    nodeShape: 'hexagon',
+    nodeType: 'source',
+  },
   transformLabel: (node, sourceTypes) =>
     `${node.name} (${sourceTypes?.find(({ id }) => id === node.source_type_id)?.product_name || node.source_type_id})`,
   children: [
     {
       name: 'service_offerings',
       label: 'Service offerings',
       attributes: ['name'],
+      topologyViewNode: {
+        nodeShape: 'square',
+        nodeType: 'serviceOfferings',
+      },
     },
     {
       name: 'service_plans',
       label: 'Service plans',
       attributes: ['name'],
+      topologyViewNode: {
+        nodeShape: 'square',
+        nodeType: 'servicePlans',
+      },
     },
     {
       name: 'service_instances',
       label: 'Service instances',
       attributes: ['name'],
+      topologyViewNode: {
+        nodeShape: 'square',
+        nodeType: 'serviceInstances',
+      },
     },
     {
       name: 'service_inventories',
       label: 'Service inventories',
       attributes: ['name'],
+      topologyViewNode: {
+        nodeShape: 'square',
+        nodeType: 'serviceInventories',
+      },
     },
     {
       name: 'service_instance_nodes',
       label: 'Service instance nodes',
       attributes: ['name'],
+      topologyViewNode: {
+        nodeShape: 'square',
+        nodeType: 'serviceInstanceNodes',
+      },
     },
     {
       name: 'service_offering_nodes',
       label: 'Service offering nodes',
       attributes: ['name'],
+      topologyViewNode: {
+        nodeShape: 'square',
+        nodeType: 'serviceOfferingsNode',
```

Comment:
@rvsia isn't there missing "s"? :)

Reply:
or one extra "s" and one missing "s"

Reply:
Yeah, but it's also missing "s" in the iconMapper so it actually works as it should. :smile:

Reply:
:mage_man: